PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java

```diff
@@ -0,0 +1,78 @@
+//Wrtite a java program to check whether an entered number is a palindrom or not
+
+//Importing scanner class inorder to use nextInt() method for taking input from user
+import java.util.Scanner;
+
+/*
+* We are given an integer input as the number, the objective is to check whether or not the given number is a palindrome.
+* To do so, we’ll first reverse the string input using loops and check if it matches the original number.
+* If it matched then the given input number is a palindrome number
+*
+* Example-1
+* -------
+* Input : 121
+* Output : This is a palindrome number
+
+* Example-2
+* -------
+* Input : 369
+* Output : This is not a palindrome number
+*
+* */
+public class Check_Palindrome {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter a number:");
+        int number = input.nextInt();// Taking input from the user
+        // If isPalindrome() method returns true then the number is palindrome and if it
+        // returns false then it is not
+        if (isPalindrome(number))
+            System.out.println("This is a palindrome number");
+        else
+            System.out.println("This is not a palindrome number");
+        input.close();
+    }
+
+    public static boolean isPalindrome(int number) {
+        /*
+         * Declaring and initializing flag variable with false
+         * if number will not palindrome then flag will return false
+         */
+        boolean flag = false;
+        /*
+         * Storing number in temp variable because
+         * after while loop number will be zero(because while loop will continue until
+         * and unless number is zero)
+         * But after while loop we need to check reverse number with original number
+         * That's why first we store the number in temp variable so that after
+         * completion of while loop
+         * we can compare temp with reverse number
+         */
+        int temp = number;
+        // Initializing rev variable with zero and after while loop the reverse of the
+        // number will store in rev
```

Comment:
Remove the comment line 53 and merge it to the line number 52 and make it single comment line.